### PR TITLE
FEATURE: Add PropertyValue node migration filter

### PR DESCRIPTION
**Pull request by kdambekalns**
Adds a new filter to match against property values in node migrations.

### CODE REVIEW

File: Neos.ContentRepository/Classes/Migration/Filters/PropertyValue.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?php
+namespace Neos\ContentRepository\Migration\Filters;
```
Comment (daniellienert) [inline Neos.ContentRepository/Classes/Migration/Filters/PropertyValue.php]:
```suggestion
declare(strict_types=1);

namespace Neos\ContentRepository\Migration\Filters;
```

File: Neos.ContentRepository/Classes/Migration/Filters/PropertyValue.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+     */
+    protected $propertyValue;
+
+    /**
+     * Sets the property name to be checked.
+     *
+     * @param string $propertyName
+     * @return void
+     */
+    public function setPropertyName($propertyName)
```
Comment (daniellienert) [inline Neos.ContentRepository/Classes/Migration/Filters/PropertyValue.php]:
```suggestion
    public function setPropertyName(string $propertyName): void
```

File: Neos.ContentRepository/Classes/Migration/Filters/PropertyValue.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        $this->propertyName = $propertyName;
+    }
+
+    /**
+     * Sets the property value to be checked against.
+     *
+     * @param string $propertyValue
+     * @return void
+     */
+    public function setPropertyValue($propertyValue)
```
Comment (daniellienert) [inline Neos.ContentRepository/Classes/Migration/Filters/PropertyValue.php]:
```suggestion
    public function setPropertyValue(string $propertyValue): void
```

**Comment by kdambekalns** [comment]
> Further it would be great to have that as DoctrineFilter which would scale much better.

For me that filter worked fine, so I was not entirely enthusiastic about changing that… but here you go. I did not test it, though. 😬

**Pull request merged**